docs/user-guide/rules.md

```diff
@@ -101,6 +101,7 @@ Don't forget to look at the list of [plugins](/docs/user-guide/plugins.md) for m
 ### Declaration block
 
 - [`declaration-block-no-duplicate-properties`](../../src/rules/declaration-block-no-duplicate-properties/README.md): Disallow duplicate properties within declaration blocks.
+- [`declaration-block-no-ignored-properties`](../../src/rules/declaration-block-no-ignored-properties/README.md): Disallow properties has no effect.
```

Comment:
How about more complete & descriptive: "Disallow properties that are ignored due to another property value in the same rule."